tests/tom-swifty-test/SwiftReflector/DynamicSelfTests.cs

```diff
@@ -210,5 +210,60 @@ public void TestSelfPropGetSet ()
 			TestRunning.TestAndExecute (swiftCode, callingCode, "Got here.\n", otherClass: auxClass, platform: PlatformName.macOS);
 
 		}
+
+
+		[Test]
+		[Ignore ("This test takes out the swift compiler when compiling wrappers")]
+		// issue opened here https://github.com/xamarin/binding-tools-for-swift/issues/374
+		public void TestSelfSubscriptGet ()
+		{
+			var swiftCode = @"
+public protocol Identity5 {
```

Comment:
Is this funky indentation required?

Reply:
Swift code in a string. Having it start fully exdented makes it easier to read.